languages/javascript/exercises/concept/array-loops/.docs/introduction.md

```diff
@@ -0,0 +1,107 @@
+<!--
+This is the markdown file with the introduction to the concept and exercise.
+
+See https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd
+-->
+
+In JavaScript, arrays are used to store multiple values in a single variable! These values can be of any type you choose, and can even be of multiple types. Arrays are also given lots of built-in methods, some of which can be used to loop over the values, giving you access to a single one on each iteration.
+
+Each of these built-in methods have a different use case, and differing syntaxes to go along with them.
+
+Looping over an array using its prototype is as easy as invoking `Array.prototype.forEach`. `forEach` takes in two parameters: a callback function and a lexical context. The callback is a function that is called every time the loop index is increased, and takes in three parameters: the current value, the current index, and the array as a whole. The second parameter to `forEach` is useful when using this method in another prototypal context (don't worry if you don't understand this right now, we'll provide some examples.)
+
+Note that because `forEach` doesn't directly expose the underlying `for` loops, you cannot use the `break` statement in the callback.
```

Comment:
```suggestion
Note that because `forEach` doesn't directly expose the underlying `for` loops, the `break` statement cannot be used in the callback.
```